### DISCUSSION TITLE
Have to install gogs server if i use more than 1 replicas

### DISCUSSION THREAD

**Discussion by nishit93-hub** [gogs/gogs#6765] [Q&A] [answered]
Hi there,

I have trying to setup gogs server  on kubernetes with 3 replicas. When i install the server from UI and try to login it ask me to install again.

I have notice that if i use more than 1 replicas then it ask to install server same as replication count time. However if i use 1 replica then it don't ask me  to install again.

**Comment by unknwon** [comment, marked as answer]
Hey, Gogs is a single stateful application, and currently does not support replicas.
